src/components/Square.js

```diff
@@ -1,17 +1,23 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React, { useState } from 'react';
 
 import './Square.css'
 
-const Square = (props) => {
+const Square = ({ id, value, onClickCallback }) => {
+
+
+  const markSquare = () => {
+    onClickCallback(id);
+  }
   // For Wave 1 enable this 
   //  Component to alert a parent 
   //  component when it's clicked on.
 
   return <button
+    onClick={markSquare}
```

Comment:
instead of creating a separate function your onClick could look like ` onClick={() => onClickCallback(id)}`